setup.py

```diff
@@ -158,7 +158,7 @@ def run_tests(self):
 
 setup(
     name='openhtf',
-    version='1.0.2',
+    version='1.0.3',
```

Comment:
+1, version bump should happen in a commit where it is the only thing.  not sure if we want to trigger a new version on this change.  is there anything else in the pipe?  might be better to get in the practice of a weekly version-bump-if-needed policy rather than when there's a commit.